tests/vm_networking_test.go

```diff
@@ -159,8 +165,44 @@ var _ = Describe("Networking", func() {
 			case "InboundVM":
 				addr = inboundVM.Status.Interfaces[0].IP
 			}
-			cmdCheck = fmt.Sprintf("ping %s -c 1 -w 5\n", addr)
+
+			// Check br1 MTU inside the pod
+			vmPod := tests.GetRunningPodByLabel(outboundVM.Name, v1.DomainLabel, tests.NamespaceTestDefault)
+			output, err := tests.ExecuteCommandOnPod(
+				virtClient,
+				vmPod,
+				vmPod.Spec.Containers[0].Name,
+				[]string{"ip", "address", "show", "br1"},
+			)
+			log.Log.Infof("%v", output)
+			Expect(err).ToNot(HaveOccurred())
+			// the following substring is part of 'ip address show' output
+			expectedMtuString := fmt.Sprintf("mtu %d", expectedMtu)
+			Expect(strings.Contains(output, expectedMtuString)).To(BeTrue())
+
+			// Check eth0 MTU inside the VM
```

Comment:
Perhaps we should name each of these checks with By(), as in https://github.com/kubevirt/kubevirt/blob/master/tests/vmlifecycle_test.go#L180 as there are several steps now.

Reply:
That can help; added in a separate commit on top.